packages/components/message-box/src/index.vue

```diff
@@ -136,8 +136,8 @@
                 :round="roundButton"
                 :disabled="confirmButtonDisabled"
                 :size="btnSize"
-                @click="handleAction('confirm')"
-                @keydown.prevent.enter="handleAction('confirm')"
+                @click="handleAction('confirm', false)"
+                @keydown.prevent.enter="handleAction('confirm', true)"
```

Comment:
According to https://github.com/element-plus/element-plus/issues/21228#issuecomment-3031774450 
consider this change:

```suggestion
                @keydown.prevent
                @keyup.enter="handleAction('confirm')"
```

It feels more natural to close the modal when releasing the enter key.

Reply:
I'm ok to use `keydown` however the current implementation fix looks quite hacky, i wonder if there is another solution 🤔 .

Reply:
has no idea